[BREAKING] Next Major Release / Port to BabylonJS (#15)
EngineHub/SchematicWebViewer: README.md

```typescript
interface SchematicRenderOptions {

>     size?: number; // Force the size of the canvas viewport, otherwise uses canvas size

Reviewer: ```suggestion
    size?: number; // Force the size of the canvas viewport, if not present use canvas size
```
